community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/setup.py

```diff
@@ -367,19 +393,50 @@ def configure_dirs():
         os.chmod(dst, 0o755)
 
 
+def _prepare_slurmrestd():
+    """
+    This is temporary workaround until
```

Comment:
I believe the image was published on Mar 17 (after Feb 28), so this hack doesn't seem to be needed anymore